src/functions/Coverage.ps1

```diff
@@ -1043,6 +1043,219 @@ function Get-JaCoCoReportXml {
     return $xml
 }
 
+function Get-CoberturaReportXml {
+    param (
+        [parameter(Mandatory = $true)]
+        $CommandCoverage,
+        [parameter(Mandatory = $true)]
+        [object] $CoverageReport,
+        [parameter(Mandatory = $true)]
+        [long] $TotalMilliseconds
+    )
+
+    if ($null -eq $CoverageReport -or ($pester.Show -eq [Pester.OutputTypes]::None) -or $CoverageReport.NumberOfCommandsAnalyzed -eq 0) {
+        return [string]::Empty
+    }
+
+    $now = & $SafeCommands['Get-Date']
+    $nineteenSeventy = & $SafeCommands['Get-Date'] -Date "01/01/1970"
+    [long] $endTime = [math]::Floor((New-TimeSpan -start $nineteenSeventy -end $now).TotalMilliseconds)
+    [long] $startTime = [math]::Floor($endTime - $TotalMilliseconds)
+
+    $folderGroups = $CommandCoverage | & $SafeCommands["Group-Object"] -Property {
+        & $SafeCommands["Split-Path"] $_.File -Parent
+    }
+
+    $packageList = [System.Collections.Generic.List[psobject]]@()
+
+    $report = @{
+        Instruction = @{ Missed = 0; Covered = 0; Total = 0 }
+        Line        = @{ Missed = 0; Covered = 0; Total = 0 }
+        Method      = @{ Missed = 0; Covered = 0; Total = 0 }
+        Class       = @{ Missed = 0; Covered = 0; Total = 0 }
+    }
+
+    foreach ($folderGroup in $folderGroups) {
+
+        $package = @{
+            Name        = $folderGroup.Name
+            Classes     = [ordered] @{ }
+            Instruction = @{ Missed = 0; Covered = 0; Total = 0 }
+            Line        = @{ Missed = 0; Covered = 0; Total = 0 }
+            Method      = @{ Missed = 0; Covered = 0; Total = 0 }
+            Class       = @{ Missed = 0; Covered = 0; Total = 0 }
+        }
+
+        foreach ($command in $folderGroup.Group) {
+            $file = $command.File
+            $function = $command.Function
+            if (!$function) { $function = '<script>' }
```

Comment:
Do we need to wrap script-content in a fake method?
I believe this allows a lines-element at class-level for root-level code.
`<!ELEMENT class (methods,lines)>`